packages/simcore-sdk/src/simcore_sdk/node_ports_v2/port.py

```diff
@@ -263,6 +263,8 @@ async def _evaluate() -> ItemConcreteValue | None:
                     io_log_redirect_cb=self._node_ports.io_log_redirect_cb,
                     r_clone_settings=self._node_ports.r_clone_settings,
                     progress_bar=progress_bar,
+                    aws_s3_cli_settings=self._node_ports.aws_s3_cli_settings,
+                    is_rclone_enabled=self._node_ports.is_rclone_enabled,
```

Comment:
question: r_clone_settings is already optional. why do we need is_rclone_enabled?

Reply:
I also thing that this should have been worded `is_s3_cli_enabled` since this is the one replacing Rclone which should be the default. Please make sure it works when the configuration is not provided.

Reply:
I have made aws_s3_cli_settings optional, if provided that it is prevered. Therefore there is no need for additional boolean env variable.